src/metatrain/__main__.py

```diff
@@ -88,6 +88,11 @@ def main():
         error_file = checkpoint_dir / error_file
 
     with setup_logging(logger, log_file=log_file, level=level):
+        logging.info(f"Package dir: {PACKAGE_ROOT}")
+        logging.info(f"Working dir: {Path('.').absolute()}")
+        logging.info(f"Version: {__version__}")
+        logging.info(f"Command line: {get_cli_input():}")
```

Comment:
Can we condense this a bit? I find that the logs are becoming quite long.
E.g. the package dir and version could be on the same line (they're quite related: e.g. "Using metatrain XXX from ...path..."
Similarly I think that the working directory and the command could belong to the same line

Reply:
Of course we can put stuff in a single line. But, for a better overview I would keep one information per line. Also, we don't gain characters if we put everything in one line. Additionally, long lines are hard to display on screens; see for example the log when we evaluate a model. If a line is super long and a line break is introduced, I find it very hard to read the log.

Reply:
Alright, fine for now, we can change it later if we notice that the logs become too messy.
I would however make these lines a bit more reader-friendly:
dir -> directory
Version -> Metatrain version
Command line -> Command-line input (perhaps something better than input, without repeating command?)